deepchecks/tabular/context.py

```diff
@@ -274,28 +285,42 @@ def task_type(self) -> TaskType:
 
     @property
     def features_importance(self) -> t.Optional[pd.Series]:
-        """Return features importance, or None if not possible."""
+        """Return feature importance, or None if not possible."""
+        # TODO: remove in future
+        get_logger().warning('"features_importance" property is deprecated use "feature_importance" instead')
+        return self.feature_importance
```

Comment:
Why not raise the DeprecationWarning?

Reply:
do you mean something like this?
```python
warnings.warn(<message>, DeprecationWarning)
```
it will not work with our `verbosity level` API,
is not it?

or do you mean something like this? 

```python
 raise DeprecationWarning(<message>)
```
this will raise an error